.gitignore

```diff
@@ -71,4 +71,6 @@ storage.test.db
 
 local-tests/build
 # local-tests/setup/networkContext.json
-.env
\ No newline at end of file
+.env
+
+packages/wrapped-keys-lit-actions/src/generated
```

Comment:
We generate all the lit action code placeholders. This are generated at build time and should not be pushed as the source of truth is the LA code itself before bundling and all that